src/dialogs/more-info-dialog.html

```diff
@@ -19,6 +19,7 @@
       }
 
       state-history-charts {
+        width: 100%;
```

Comment:
I'm down to do a min-width on the dialog. Not sure about a max-width because for things like the camera we do want it to be as big as possible.